docker/compose.yml

```diff
@@ -0,0 +1,143 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+services:
+  minio:
+    image: bitnami/minio:latest
```

Comment:
minio also does not have an arm64 Docker image...

Reply:
MinIO itself should support ARM64[1], I don't know if bitnami or other vendors provide ARM64 docker image, maybe we can build it from scratch.

[1] https://dl.min.io/server/minio/release/

Reply:
The `minio/minio`[1] image support arm64

[1] https://hub.docker.com/r/minio/minio

Reply:
Yes, this is what I used too! (https://github.com/apache/incubator-kyuubi/issues/2365#issuecomment-1099960996)